ee-9/feature-pack/pom.xml

```diff
@@ -517,6 +517,17 @@
                     <groupId>${project.groupId}</groupId>
                     <artifactId>jipijapa-spi</artifactId>
                 </exclusion>
+                
+                <exclusion>
+                    <groupId>${ee.maven.groupId}</groupId>
+                    <artifactId>wildfly-opentelemetry-api</artifactId>
+                </exclusion>
+
+                <exclusion>
+                    <groupId>${ee.maven.groupId}</groupId>
+                    <artifactId>wildfly-opentelemetry</artifactId>
+                </exclusion>
```

Comment:
Please move these two below, in the right spot alphabetically with the others.

Reply:
lol. Wow. I thought I had. :P Moved.